draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -213,15 +213,17 @@ querying via recursive resolvers.
 
 # Properties {#properties}
 
-Member zones may optionally be assigned additional properties represented by
-RRsets below the corresponding member node. This document suggests a few
-properties which are all optional to implement.
+Each member zone MAY have one or more of additional properties, described in this chapter.
+These properties are completely optional and the catalog zone consumer SHOULD ignore those it doesn't understand.
+Each property is defined by a zone record in a sub-node of the node, in which the member-defining PTR record resides.
 
 ## The Group Property
 
-The group property allows to add at most one tag per member zone.
-The consumer might handle/configure member zones differently based on the tag.
-The group property is represented by a TXT Resource Record, for example:
+The property is defined by a TXT record in the sub-node labelled `group`.
+The value of the TXT record SHOULD be at most 255 octets long and SHOULD NOT contain whitespace characters.
```

Comment:
I'm fine with making this MUST, both times.

Reply:
Resolverd